apps/src/templates/studioHomepages/SectionsAsStudentTable.jsx

```diff
@@ -52,33 +75,41 @@ class SectionsAsStudentTable extends React.Component {
       >
         <thead>
           <tr style={styles.headerRow}>
-            <td
+            <th
```

Comment:
These cells are all in the header row so they should be `th` tags, this also helps make styling easier